Subject: [xf86-video-vesa] add missing Xproto.h
From: Robert Schwebel <r.schwebel@pengutronix.de>

When cross compiling xf86-video-vesa-2.2.1, I get this error:

 i586-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I.. -I/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg -I../src -I/usr/include -isystem /home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/include -isystem /home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include -g -O2 -I/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg -I/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/pixman-1 -I/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include -I../src -MT vesa.lo -MD -MP -MF .deps/vesa.Tpo -c vesa.c  -fPIC -DPIC -o .libs/vesa.o
In file included from /home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg/dix.h:55,
                 from /home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg/scrnintstr.h:58,
                 from /home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg/xf86str.h:39,
                 from /home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg/xf86.h:47,
                 from vesa.h:36,
                 from vesa.c:48:
/home/rsc/svn/oselas/bsp/pengutronix/OSELAS.BSP-Pengutronix-AllYes-trunk/platform-i586/sysroot-target/usr/include/xorg/geext.h:59: error: expected ')' before '*' token

The packet forgets to include Xproto.h, which is where xGenericEvent is defined.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 src/vesa.c |    1 +
 1 file changed, 1 insertion(+)

Index: xf86-video-vesa-2.2.1/src/vesa.c
===================================================================
--- xf86-video-vesa-2.2.1.orig/src/vesa.c
+++ xf86-video-vesa-2.2.1/src/vesa.c
@@ -44,6 +44,7 @@
 #endif
 
 #include <string.h>
+#include <X11/Xproto.h>
 
 #include "vesa.h"
 
